ibazel/ibazel_test.go

```diff
@@ -205,109 +220,64 @@ func TestIBazelRun_firstPass(t *testing.T) {
 	}
 
 	mockBazel.AssertActions(t, expected)
+}
 
-	if cmd.Stdout != os.Stdout {
-		t.Errorf("Didn't set Stdout correctly")
-	}
-	if cmd.Stderr != os.Stderr {
-		t.Errorf("Didn't set Stderr correctly")
-	}
-	if cmd.SysProcAttr.Setpgid != true {
-		t.Errorf("Never set PGID (will prevent killing process trees -- see notes in ibazel.go")
-	}
-
-	if err := cmd.Wait(); err != nil {
-		t.Errorf("Subprocess was never started. State: %v, Err: %v", cmd.ProcessState, err)
+func TestIBazelRun_notifyPrexistiingJobWhenStarting(t *testing.T) {
+	oldDefaultCommand := commandDefaultCommand
+	commandDefaultCommand = func(b bazel.Bazel, target string, args []string) command.Command {
```

Comment:
commandDefaultCommand is a very confusing name to me. Can't you just call this createMockCommand or something along those lines?

Reply:
commandDefaultCommand is the dot path reference to the package and function that is being mocked out. If I name it "createMockCommand" then in ibazel.go on line 256 it will be extremely confusing because it is creating the real thing.

Reply:
I misunderstood. Carry on